pkg/csi/service/wcp/controller.go

```diff
@@ -835,7 +854,12 @@ func (c *controller) createFileVolume(ctx context.Context, req *csi.CreateVolume
 	}
 	volumeID, faultType, err = common.CreateFileVolumeUtil(ctx, cnstypes.CnsClusterFlavorWorkload, vc,
 		c.manager.VolumeManager, c.manager.CnsConfig, &createVolumeSpec, filteredDatastores,
-		filterSuspendedDatastores, isTKGSHAEnabled, checkCompatibleDataStores)
+		filterSuspendedDatastores, isTKGSHAEnabled,
+		cnsvolume.CreateVolumeExtraParams{
```

Comment:
Do we support creating file volume on stretched supervisor cluster?

Reply:
We do not support it on stretch supervisor. Removed the extraParams here and sending nil value.